corpus/mets-xml/mets/OBJID/testCase.xml

```diff
@@ -0,0 +1,141 @@
+<testCase testable="TRUE">
+  <id specification="CSIP" version="2.0.0-DRAFT" requirementId="CSIP1" />
+  <references>
+    <reference requirementId="CSIP1" URL="http://earkcsip.dilcis.eu#CSIP1">
+      | ID | Name &amp; Location | Description &amp; usage | Cardinality &amp; Level |
+      | -- | --------------- | ------------------- | ------------------- |
+      | **CSIP1** | **Content Identification** `mets/@OBJID` | It is mandatory to use a content ID which is expressed with @OBJID. The value should be the same as the name or ID of the package (the name of the root folder) for the root METS document or the name and folder name for the representation. The OBJID must meet the principle of being unique at least across the repository. | **1..1** MUST |
+    </reference>
+    <reference requirementId="the-use-of-identifiers" URL="http://earkcsip.dilcis.eu/#the-use-of-identifiers">
```

Comment:
I am not sure the ID requirement goes for OBJID. I understand the ID requirement as only for XML-datatype "ID".